script.js

```diff
@@ -0,0 +1,34 @@
+const hambutton = document.getElementById('menu-harmbuger');
+const main = document.querySelector('.container');
+const menu = document.querySelector('.modal-container');
+
+const hamburgerTwo = document.getElementById('modal-harmburger');
+
+function openMenu() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer `arrow functions ` over normal functions, so as to keep your _function's_ syntax simpler and cleaner.